src/main/java/dev/isxander/pronounmc/mixins/SocialInteractionsPlayerListEntryMixin.java

```diff
@@ -0,0 +1,31 @@
+package dev.isxander.pronounmc.mixins;
+
+import dev.isxander.pronounify.utils.MessageManager;
+import dev.isxander.pronounify.utils.PronounManager;
+import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.gui.screen.multiplayer.SocialInteractionsPlayerListEntry;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import java.util.UUID;
+
+@Mixin(SocialInteractionsPlayerListEntry.class)
+public class SocialInteractionsPlayerListEntryMixin {
+
+    @Shadow @Final private UUID uuid;
+
+    @Redirect(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/font/TextRenderer;draw(Lnet/minecraft/client/util/math/MatrixStack;Ljava/lang/String;FFI)I"))
```

Comment:
Can you use a `@ModifyArg` instead of a redirect?

Reply:
Done.

Reply:
Ah, I see why this was done now. Colours no longer work.